pkg/status/metrics.go

```diff
@@ -13,6 +13,7 @@ const (
 
 var phaseToGaugeValue = map[v1.HelmReleasePhase]float64{
 	// Unknown is mapped to 0
+	v1.HelmReleasePhaseUninstallFailed:  -5,
```

Comment:
Given there has not been a release yet with this new metric, I would reorder the integers so that they make sense again (`ChartFetchFailed` being the lowest negative integer).

Reply:
Gotcha makes sense.